pallets/gear/src/weights.rs

```diff
@@ -1226,6 +1231,64 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
             // Standard Error: 8_255
             .saturating_add(Weight::from_ref_time(1_114_949).saturating_mul(r.into()))
     }
+
+    // FIXME: use the result of `instr_i64extendsi32` now
+    // waiting for the next generation.
+    fn instr_i32extend8s(r: u32, ) -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `0`
+        //  Estimated: `0`
+        // Minimum execution time: 1_900 nanoseconds.
+        Weight::from_ref_time(1_701_470)
```

Comment:
We recently removed all use cases for `Weight::from_ref_time(ref_time)` as it is deprecated in master branch
Please replace it with `Weight::from_parts(ref_time, 0)`.

I just found that .hbs file isn't modified
![image](https://user-images.githubusercontent.com/109800286/233917701-dbef9b19-65df-478f-bdca-976f66c8a6ef.png)

Reply:
These weights can be generated later using the updated template.  We just need to make sure the template is ok.

Reply:
I think we must remove this weight from template - because it's contains really hard template code and useless, because we can calculate this constant in scheduler.